packages/react/src/components/radio-group/radio-group.tsx

```diff
@@ -30,14 +32,23 @@ function RadioGroupInternal<T>(props: RadioGroupProps<T>) {
   return (
     <RadioGroupContext.Provider value={radioGroupCore}>
       <div>
-        {props.children}
+        {props.name
+          ? Children.map(
+            props.children,
+            child => (
+              isValidElement<EditorProps<T>>(child)
+                ? cloneElement(child, { name: props.name })
+                : child
```

Comment:
**Question**:
Could we use context-based approach here? So that children get `name` from RadioGroupContext?

Reply:
Yes, but in this case, we'll need to store the name in the core as well